pytd/writer.py

```diff
@@ -17,13 +17,27 @@
 
 
 def _to_list(ary):
+    is_null_ary = pd.isnull(ary)
+    if ary is None or (not isinstance(is_null_ary, np.ndarray) and is_null_ary):
+        return None
```

Comment:
Can you elaborate what these lines are doing? I couldn't fully understand the point, and your up-to-date unit tests actually succeed even if we delete them..

Reply:
refactored d97e189